packages/@aws-cdk/aws-cloudwatch/lib/dashboard.ts

```diff
@@ -107,15 +115,19 @@ export class Dashboard extends Resource {
       }
     }
 
+    if (props.start !== undefined && props.defaultInterval !== undefined) {
```

Comment:
```suggestion
    if (props.start && props.defaultInterval) {
```

Reply:
In some cases it is safe to omit `!== undefined`, and in some cases it leads to bugs (for example, in the case of nullable integers, and occasionally in the case of nullable strings depending on what you want to do with the string).

When you're not *super* experienced yet, it's probably safer to always write `!== undefined`. So I support the current use.